Backend/wrappers/standardAPI.py

```diff
@@ -0,0 +1,33 @@
+import tweepy
+
+import wrappers.credentials as creds
+
+
+class standardAPI:
+
+    def __init__(self):
+        self.api = creds.create()
+
+
+    def retrieve_trends(self, woeid, num):
+        query = self.api.trends_place(woeid)
+        top_trends  = self.query_transform(query, woeid, num)
+        return top_trends
+
+
+    def getTrendsClose(self, lat, long, num):
+        query = self.api.trends_closest(lat, long)
+        return query
+    
+
+    def query_transform(self, json_result, woeid, num):
+        trendsResult = json_result[0]
+        place = trendsResult['locations'][0]['name']
+        trends_list = trendsResult['trends']
+        trends = []
+        for i in range(0, num):
+            trends.append(trends_list[i]['name'])
```

Comment:
it would be really useful, especially for the temporal analysis feature, if this also returned the 'tweet_volume' attribute of the trend result entries along with the trends' names. Would you please add that or just put a "#TODO" comment here about that?